acapi/resources/backup.py

```diff
@@ -28,13 +28,15 @@ def delete(self):
         self.request(method='DELETE')
         return True
 
-    def download(self, target_file):
+    def download(self, target_file, stream=True):
```

Comment:
Sorry my last comment on this was unclear. I think we can remove the option for streaming and always stream. It simplifies the code and makes writes more reliable.